src/libs/actions/User.js

```diff
@@ -65,6 +65,56 @@ function getBetas() {
     });
 }
 
+/**
+ * Fetch the public domain info for the current user (includes secondary logins).
+ *
+ * This API is a bit weird in that it sometimes depends on information being cached in bedrock.
+ * If the info for the domain is not in bedrock, then it creates an asynchronous bedrock job to gather domain info.
+ * If that happens, this function will automatically retry itself in 10 minutes.
+ *
+ * @param {String} loginList
+ */
+function getDomainInfo(loginList) {
+    // If this command fails, we'll retry again in 10 minutes,
+    // arbitrarily chosen giving Bedrock time to resolve the ClearbitCheckPublicEmail job for this email.
+    const RETRY_TIMEOUT = 600000;
```

Comment:
NAB: Could use a numeric seperator here:
```suggestion
    const RETRY_TIMEOUT = 600_000;
```

Reply:
Ah, not your code anyway

Reply:
TIL about numeric separators